examples/configs/user_settings_arduino.h

```diff
@@ -25,7 +25,10 @@
 */
 
 /* Define a macro to display user settings version in example code: */
-#define WOLFSSL_USER_SETTINGS_ID "Arduino user_settings.h v5.7.4"
+#define WOLFSSL_USER_SETTINGS_ID "Arduino user_settings.h v5.7.6"
+
+/* Disable wolfcrypt cryptographic security hardening. Comment out to enable: */
+#define WC_NO_HARDEN
```

Comment:
I don't support any examples that define this... Hardening should always be enabled unless the build is public operations only.